packages/react-wildcat-handoff/src/utils/defaultTemplate.js

```diff
@@ -48,7 +48,7 @@ module.exports = function defaultTemplate(cfg) {
         ` : `
         <script>
             // Remove any registered service workers (dev mode only)
-            if ("serviceWorker" in navigator) {
+            if ("serviceWorker" in navigator && location.protocol === "https:") {
```

Comment:
Hm, you have access to `protocol` in [`wildcat.config.js`](https://github.com/nfl/react-wildcat/blob/master/example/wildcat.config.js#L151). Might be better to check for this at app runtime instead of client runtime.